modules/table/src/main/java/org/apache/ignite/internal/table/distributed/TableManager.java

```diff
@@ -861,9 +876,10 @@ private RaftGroupOptions groupOptionsForPartition(
                 raftMgr.topologyService(),
                 //TODO IGNITE-17302 Use miniumum from mv storage and tx state storage.
                 new OutgoingSnapshotsManager(raftMgr.messagingService()),
-                partitionStorage::persistedIndex,
+                new MvPartitionAccess(tableStorage, partId),
```

Comment:
Please coordinate with @rpuch on these changes to reduce the severity of merge conflicts

Reply:
Discussed.